5calls/app/src/main/java/org/a5calls/android/a5calls/view/OutcomeView.java

```diff
@@ -0,0 +1,105 @@
+package org.a5calls.android.a5calls.view;
+
+import android.content.Context;
+import android.content.res.Configuration;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.AppCompatButton;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.LinearLayout;
+
+import org.a5calls.android.a5calls.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class OutcomeView extends LinearLayout {
+
+    public static final String VM = "vm";
+    public static final String VOICEMAIL = "voicemail";
+    public static final String CONTACT = "contact";
+    public static final String CONTACTED = "contacted";
+    public static final String UNAVAILABLE = "unavailable";
+
+    private List<Button> buttons;
+
+    public OutcomeView(Context context, String[] outcomes, final Callback callback) {
+        super(context);
+
+        setLayoutParams(new LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
+        setOrientation(VERTICAL);
+
+        if (outcomes != null && outcomes.length > 0) {
+            int orientation = context.getResources().getConfiguration().orientation;
+            if (orientation == Configuration.ORIENTATION_PORTRAIT) {
+                setOrientation(VERTICAL);
+            } else {
+                setOrientation(HORIZONTAL);
+            }
+
+            buttons = new ArrayList<>();
+            for (final String outcome : outcomes) {
+                Button button = new AppCompatButton(context);
+                button.setMinHeight(dpsToPixels(context, 48));
```

Comment:
Please use R.dimen.accessibility_min_size in dimens.xml rather than hard-coding values.

Reply:
Fixed in new RecyclerView item layout. See:

https://github.com/5calls/android/pull/89/files?diff=split#diff-6d9976551a58cdfcf44f3b6a3c33c12aR6